### PR TITLE
Link paragraph: optionally display as a button style

### PR DESCRIPTION
**Pull request by sonianb**
Relates #92 

- Add an option on the Link paragraph to display as a link or a button style. 
- On the link paragraph type, add a select field named Link style, with the option button|Button and field description "Choose the style of the link".
- Import the changes.

### CODE REVIEW

**Comment by markconroy** [comment]
1. Should there be any CSS with this to style the link/button?
2. Should there be any change to a template to facilitate this?

**Comment by markconroy** [comment]
Ignore me, I see it's in the localgov_base repo.

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml [hunk 1]
```diff
@@ -2,28 +2,39 @@ langcode: en
...
+    type: list_key
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 2
+    region: content
   localgov_title:
-    weight: 0
+    type: string
     label: above
```
Comment (markconroy) [inline config/install/core.entity_view_display.paragraph.localgov_link.default.yml]: I think we should remove the label from above here, it's jarring on the backend when editing the CMS.

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml [hunk 2]
```diff
@@ -2,28 +2,39 @@ langcode: en
...
     settings:
       link_to_entity: false
     third_party_settings: {  }
-    type: string
+    weight: 0
     region: content
   localgov_url:
-    weight: 1
+    type: string
     label: above
```
Comment (markconroy) [inline config/install/core.entity_view_display.paragraph.localgov_link.default.yml]: I think we should remove the label from above here, it's jarring on the backend when editing the CMS.

**Review by markconroy — CHANGES_REQUESTED** [review]

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml (hunk 1 above)
Comment (sonianb) [reply config/install/core.entity_view_display.paragraph.localgov_link.default.yml]: This has now been removed.

**Review by markconroy — APPROVED** [review]

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml (hunk 2 above)
Comment (sonianb) [reply config/install/core.entity_view_display.paragraph.localgov_link.default.yml]: This one has been removed as well.

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml [hunk 3]
```diff
@@ -2,28 +2,37 @@ langcode: en
...
+    type: list_key
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 2
+    region: content
   localgov_title:
-    weight: 0
-    label: above
+    type: string
```
Comment (msayoung) [inline config/install/core.entity_view_display.paragraph.localgov_link.default.yml:24]: we need `label: hidden` in here

File: config/install/core.entity_view_display.paragraph.localgov_link.default.yml [hunk 4]
```diff
@@ -2,28 +2,37 @@ langcode: en
...
     settings:
       link_to_entity: false
     third_party_settings: {  }
-    type: string
+    weight: 0
     region: content
   localgov_url:
-    weight: 1
-    label: above
+    type: string
```
Comment (msayoung) [inline config/install/core.entity_view_display.paragraph.localgov_link.default.yml:32]: we need `label: hidden` in here

**Review by msayoung — CHANGES_REQUESTED** [review]

**Comment by msayoung** [comment]
We're getting a warning 
`Warning: Undefined array key "label" in Drupal\Core\Field\Plugin\Field\FieldFormatter\StringFormatter::create() (line 73 of core/lib/Drupal/Core/Field/Plugin/Field/FieldFormatter/StringFormatter.php). ` on /admin/structure/paragraphs_type/localgov_link/display

We need to add the label config back in to config/install/core.entity_view_display.paragraph.localgov_link.default.yml , but change it to `label: hidden`

**Review by msayoung — APPROVED** [review]

**Pull request merged**